server.js

```diff
@@ -244,9 +243,9 @@ app.post("/", async (req, res) => {
 
         const baseUrl = `${req.protocol}://${req.get("host")}/${BASE_URI}`;
         const params = new URLSearchParams(req.body).toString();
-        const src = baseUrl + path + "?" + params;
+        const src = `${baseUrl + path}?${params}`;
```

Comment:
```suggestion
        const src = `${baseUrl}${path}?${params}`;
```

Reply:
[${} - MDN doc](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#expression_interpolation) indicates Template literals can be used for expressions.

```
let a = 5;
let b = 10;
console.log(`Fifteen is ${a + b} and
not ${2 * a + b}.`);
// "Fifteen is 15 and
// not 20."
```

Reply:
I agree for arithmetic expressions but here, it's just string concatenation so since you are already using a string literal, I don't see the point in using the `+` operator.